docs/content/latest/admin/yb-admin.md

```diff
@@ -1515,6 +1515,38 @@ yb-admin \
     list_cdc_streams
 ```
 
+#### delete_cdc_stream <stream_id> [force_delete]
+
+Deletes underlying CDC stream for the specified YB-Master servers.
+
+**Syntax**
+
+```sh
+yb-admin \
+    -master_addresses <master-addresses> \
+    delete_cdc_stream <stream_id [force_delete]>
+```
+
+* _master-addresses_: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
```

Comment:
As a side note, I feel that there can be some confusion for these xcluster-related yb-admin commands as to which cluster's `master-addresses` to use, but don't really know of the best way to clarify (maybe its enough to explain in the setup pages with examples?)

Reply:
This might deserve an overhaul as to how we reference these across all commands.